geonode/harvesting/harvesters/geonode.py

```diff
@@ -263,10 +270,14 @@ def finalize_resource_update(
             GeoNodeResourceType.DOCUMENT.value
         )
         if is_document:
-            geonode_resource.thumbnail_url = (
-                harvested_info.resource_descriptor.distribution.thumbnail_url)
-            geonode_resource.doc_url = (
-                harvested_info.resource_descriptor.distribution.original_format_url)
+            if len(harvested_info.copied_resources) > 0:
+                # TODO: generate a thumbnail for the local document
```

Comment:
the `resource_manager.ser_thumbnail` method should do that already

Reply:
OK, I'll amend this